terraform/parameter_store.tf

```diff
@@ -2,26 +2,34 @@ resource "aws_ssm_parameter" "wordpress-db-host" {
   name        = "/wordpress/WORDPRESS_DB_HOST"
```

Comment:
you should use the value that correspond to the secret.. in that case the db-host